gatsby-node.js

```diff
@@ -19,6 +19,7 @@ module.exports.modifyWebpackConfig = ({ config, program }) => {
         resolve: {
             root: path.join(__dirname, 'src'),
         },
+        target: 'web',
```

Comment:
This can be omitted as it's the default value (https://webpack.js.org/configuration/target/#string).

On the other hand, this way it's more explicit, so I'll leave it up to you.

Reply:
It should be the default. However, gatsby sets `node` as the default [here](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/webpack.config.js#L560).  This was the line introduced to solve the problem 😛

Reply:
Have mercy, my bad 🤦‍♂️

Good job 👍